src/main/java/com/nike/cerberus/operation/core/UpdateStackOperation.java

```diff
@@ -191,7 +180,8 @@ public boolean isRunnable(final UpdateStackCommand command) {
         }
 
 
-        final TypeReference<Map<String, String>> typeReference = new TypeReference<Map<String, String>>() {};
+        final TypeReference<Map<String, String>> typeReference = new TypeReference<Map<String, String>>() {
```

Comment:
nit: I know IntelliJ autoformats the closing brace to the next line, but I've been putting them on the same line

Reply:
Done